builtin/plugins/dposv3/dpos.go

```diff
@@ -197,6 +198,40 @@ func (c *DPOS) Init(ctx contract.Context, req *InitRequest) error {
 		params.DowntimePeriod = defaultDowntimePeriod
 	}
 
+	candidates, err := LoadCandidateList(ctx)
+	if err != nil {
+		return err
+	}
+
+	// if InitCandidates is true, whitelist validators and register them for candidates
+	if params.InitCandidates {
```

Comment:
`params` are persisted in the `State`, there's no reason for this to be in there though.

Reply:
Moved it to `DPOSInitRequest` instead